### PR TITLE
change_virtio_mem_request_size:Fix parameter matching issue

### PR DESCRIPTION
**Pull request by zhenyzha**
Fix parameter matching issue, add % to format the string.

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
@Yingshun Could you pls help review when you are free. thanks.
` (1/1) type_specific.io-github-autotest-libvirt.memory.devices.virtio_mem.change_memory.file.normal_requested.running_guest: PASS (168.37 s)`

File: libvirt/tests/src/memory/memory_devices/change_virtio_mem_request_size.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def check_various_size(test, vm_name, expected_size,
         test.fail("Expect to get '%s':'%s', but got:'%s' in '%sth' "
                   "memory" % (check_item, str(expected_size), actual_size, index+1))
     else:
-        test.log.debug("Check '%s' is '%s' successfully", (check_item, expected_size))
+        test.log.debug("Check '%s' is '%s' successfully" % (check_item, expected_size))
```
Comment (Yingshun) [inline libvirt/tests/src/memory/memory_devices/change_virtio_mem_request_size.py:110]:
```suggestion
        test.log.debug("Check '%s' is '%s' successfully" , check_item, expected_size)
```

Comment (zhenyzha) [reply libvirt/tests/src/memory/memory_devices/change_virtio_mem_request_size.py]: Corrected, thanks for your suggestion.

**Comment by zhenyzha** [comment]
@Yingshun Could you pls help review again when you are free? thanks.

` (1/1) type_specific.io-github-autotest-libvirt.memory.devices.virtio_mem.change_memory.file.normal_requested.running_guest: PASS (172.40 s)`

**Review by Yingshun — APPROVED** [review]

**Pull request merged**